dev/devicelab/lib/framework/adb.dart

```diff
@@ -237,6 +246,11 @@ class AndroidDevice implements Device {
     await shellExec('input', const <String>['keyevent', '82']);
   }
 
+  @override
+  Future<Null> tap(int x, int y) async {
+    await shellExec('input', <String>['tap', '$x', '$y']);
```

Comment:
We seem to test these kind of things in `adb_test.dart`, while I agree there isn't much to test I think we should be consistent? (test this as well, or not test things like togglePower)

Reply:
excellent catch. added test for `tap`.